lib/src/view/home/home_tab_screen.dart

```diff
@@ -749,20 +749,33 @@ class _GamePreview<T> extends StatelessWidget {
 }
 
 class _RelationButton extends ConsumerWidget {
-  const _RelationButton();
+  const _RelationButton(this.wasOnline);
```

Comment:
The `wasOnline` is weird here. We want to disable the button if currently not online, the `wasOnline` serves another purpose.